README.md

```diff
@@ -1,13 +1,23 @@
 # Project Mongo API
 
-Replace this readme with your own information about your project.
+✓ What MongoDB is
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+✓ How to model data in Mongo using Mongoose
+
+✓ How to store secret information such as database passwords
+
+✓ How to return useful error messages from your API endpoints
+
+✓ How to fetch items from a Mongo database using Mongoose
+
+✓ How to seed large amounts of data to a database
 
 ## The problem
 
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://project-mongo-api-j6bcvtd3pq-lz.a.run.app/
```

Comment:
Nice that you added the text here on the first page - I'm curious how you solved that one?
Mine just goes straight to the API and not with any explanation... Nice solution!